app_dart/lib/src/service/scheduler/policy.dart

```diff
@@ -64,8 +64,8 @@ class BatchPolicy implements SchedulerPolicy {
     // Ensure task isn't considered in recentTasks
     recentTasks.removeWhere((Task t) => t.commitKey == task.commitKey);
     if (recentTasks.length < kBatchSize) {
-      log.warning('${task.name} has less than $kBatchSize, triggerring all builds regardless of policy');
-      return LuciBuildService.kDefaultPriority;
+      log.warning('${task.name} has less than $kBatchSize, skip scheduling to wait for ci.yaml roll.');
+      return null;
```

Comment:
This is the main change.

Reply:
qq: What does null represent in terms of scheduling?

Reply:
`null` here means no schedule. See https://github.com/flutter/cocoon/blob/c93ec99b2c0d40f410368cfad2d62e2ca19608c5/app_dart/lib/src/service/scheduler.dart#L145